src/lando/settings.py

```diff
@@ -48,6 +48,9 @@
 SESSION_COOKIE_SECURE = True
 CSRF_COOKIE_SECURE = True
 
+# Set the upload size to 10x the default value (2.5MiB); see bug 1964604.
+DATA_UPLOAD_MAX_MEMORY_SIZE = 26214400
```

Comment:
It wouldn't hurt to treat this like other variables, i.e., have a default but try to fetch from the environment.

Reply:
Very good point!